parts/k8s/containeraddons/azure-policy-deployment.yaml

```diff
@@ -0,0 +1,595 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    controller-tools.k8s.io: "1.0"
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
```

Comment:
Do we definitely want to ship this addon without any user-configuration vector? Setting to `Reconcile` means that in order update the config a user will have to manually log onto all master nodes and update the spec @ `/etc/kubernetes/addons/`. Is that what we want?

Reply:
gatekeeper `config` is set as `ensureExists` and variables configurable through addon config. i think we are okay for reconcile for rest.